### ISSUE TITLE
Getting the Most out of Github Issues Guide

### ISSUE THREAD

**Issue by alexandrastubbs** [hackforla/website#794]
### Dependency
New Issue for figuring out how we can use markdown and then this issue will be restructured. #1085

### Overview 

The 'Getting the most out of Github Issues' guide is complete, but not live due to a problem with how it is showing up visually. We need to fix the visuals so that we can get it live in our toolkit. 

### Action Items 
Development 
- [x] Discuss with Danielle how the CSS should be set up 
    - [ ] If restructuring is recommended, then fix the survey-reporting-dashboard-guide to use new SCSS format and do PR 
    - [x] Update the getting the most out of github guide to use the new SCSS format 
- [x] Add link to the toolkit page and update the card to reflect live status 
- [ ] Create a new Wiki page with a clear explanation on how to execute the responsiveness of the red link indicators and link it to the architecture/ how to create a guide  

### Resources 
[Github Issue Guide](https://www.hackforla.org/guide-pages/github-issues.html)
[Example of guide for what it should look like](https://www.hackforla.org/guide-pages/survey-reporting-dashboard-guide)
[Toolkit page](https://www.hackforla.org/toolkit)
[Original issue for the guide](https://github.com/hackforla/website/issues/514)
[How to create a guide page](https://github.com/hackforla/website/wiki/Hack-for-LA's-Site-Architecture#guide-pages)

Image highlighting the problem: 
<img width="1231" alt="Screen Shot 2020-10-23 at 11 47 06 AM" src="https://user-images.githubusercontent.com/66918724/97036779-9011f000-1525-11eb-800c-4ea2281f64f4.png">

**Comment by alexandrastubbs** [comment]
Hannah Livnat was the developer for the Survey Reporting Dashboard page btw, in case you'd like to understand what she did to make that page work.

**Comment by ExperimentsInHonesty** [comment]
**Notes from a slack convo with @hannahlivnat who can be contacted on Slack at Hannah Livnat
Hannah said: ...I  made the guide arrows as reusable sass mixins within the main github issue guide scss page so any developer should be able to achieve the correct affect by using the correct mixins and class names.  https://github.com/hackforla/website/blob/gh-pages/_sass/components/_guide-pages.scss**

**class name for any div containing image with overlay links:**
![image](https://user-images.githubusercontent.com/37763229/100830670-081ee000-3419-11eb-9e24-1cbaf436d963.png)

**Class name for image within overlay-link div:**
![image](https://user-images.githubusercontent.com/37763229/100830715-208efa80-3419-11eb-8c79-574135f1e2a2.png)

**Mixin for small and medium arrows:**
![image](https://user-images.githubusercontent.com/37763229/100830757-3a304200-3419-11eb-9747-08f29949d5d5.png)

**classname for arrows that should be hidden on mobile:**
![image](https://user-images.githubusercontent.com/37763229/100830794-52a05c80-3419-11eb-8da8-736736caa7bc.png)

**mixin for arrows that should be changed on mobile:**
![image](https://user-images.githubusercontent.com/37763229/100830839-6d72d100-3419-11eb-891e-61ca0689804a.png)

**Comment by pawan92** [comment]
@alyssabenipayo Please update your issue with following information
1. Progress
2. Blockers
3. Availability
4. ETA

**Comment by abenipa3** [comment]
Hello @pawan92, this is my progress so far.

I updated the HTML and SCSS files for Getting the Most out of GitHub Guide as seen below so that the arrow points are positioned properly on full screen. Restructuring was not needed. 

(Full Screen Version Preview)
![image](https://user-images.githubusercontent.com/38295612/104141715-f2cc9800-536c-11eb-98e9-2d22dd481fda.png)
(Mobile Version Preview)
![image](https://user-images.githubusercontent.com/38295612/104141786-52c33e80-536d-11eb-9bd2-fb4dd8488747.png)

I also added the link to the toolkit page and updated the card to reflect live status.
![image](https://user-images.githubusercontent.com/38295612/104142163-2e686180-536f-11eb-8226-78f28a874072.png)

**Comment by abenipa3** [comment]
- Progress: Approximately 85% done with updating the Getting the Most out of Github Guide. 
- Blocks: Need to figure out how to resolve the visual issue when viewing in iPad mode. (UPDATE 1/14+1/16: Reached out to Hannah for questions.)
- Availability: Open
- ETA: 1/17 for the Getting the Most out of Github Guide. Wiki Page TBD

**Comment by abenipa3** [comment]
UPDATE 1/19/21
- Progress: ~Visual issue resolved - Created a pull request. Will start on writing the wiki page.~ Still need to resolve visual issue (this time on mobile).
- Blocks: Arrows are pushed on top of the elements. Experimenting with media query so that it will work on all devices.
- Availability: Open
- ETA: GitHub Issues, hopefully, 1/20/21...Wiki Page - TBD

**Comment by pawan92** [comment]
@alyssabenipayo Could you update please
Progress: 
Blocks: 
Availability: 
ETA:

**Comment by abenipa3** [comment]
Progress: Reformatting the github-issues-guide.scss so that it would use the classes from the _guide-pages.scss 
Blocks: Need to stabilize arrows so that it doesn't move on the overlay image on mobile devices.
Availability: Open
ETA: GitHub Issues - 1/30...Wiki Page - TBD

**Comment by abenipa3** [comment]
Progress: Stabilized arrows. Need to resize text.
Blocks: Text spreads over the grey area as seen below. Still experimenting with font-size till the issue resolves.
Availability: Open on Monday, Tuesday, and Wednesday.
ETA: GitHub Issues 2/2. Wiki Page - TBD
![image](https://user-images.githubusercontent.com/38295612/106379587-d1eed580-6361-11eb-81af-a721328ce36e.png)

**Comment by abenipa3** [comment]
Progress: Started working on the Wiki Page while figuring out how to fix the text sizes via mobile.
Blocks: Text overlaps the grey area on mobile. Wasn't feeling well this past week as well, but I'm good now and looking forward to progressing further with this issue.
Availability: Open on Monday through Thursday.
ETA: GitHub Issues 2/9/21. Wiki Page TBD.
![image](https://user-images.githubusercontent.com/38295612/107143002-6d92bf80-68e7-11eb-91e6-d17e2caf8bb4.png)

Link to Wiki: https://docs.google.com/document/d/1dPlPiNLRaueoSPbcgBwdHLSHMaNeyeFNKwUaBLMiIpo/edit
![image](https://user-images.githubusercontent.com/38295612/107155878-d2720800-692f-11eb-9bf9-a1b71922a5b7.png)

**Comment by daniellex0** [comment]
@alyssabenipayo This issue has us adding more content to this page: issue #918 

You can finish the issue as-is and then leave it to someone else, or you can work this into your changes- it's up to you

**Comment by abenipa3** [comment]
@daniellex0 Thanks for the update. I think I can work this into my changes. If anything, I'll keep everyone posted. Thanks.

**Comment by abenipa3** [comment]
Progress: Code works well in Firefox for Getting the Most out of GitHub as seen below, but not Google Chrome. Also attempting to resolve visual issues in the Survey Reporting file as seen below and discovered that outputs are not cross-browser compatible.

Blocks: Need to figure out how to make it cross-browser compatible. Also may consider restructuring and fixing the survey-reporting-dashboard-guide, but want to check in with the team if there are other suggestions.

Availability: Open.

ETA: GitHub Issues 2/20/21 (It may take longer considering the complexity). Wiki Page TBD.

Firefox Output (iPhone 6/7/8 Preview)
![image](https://user-images.githubusercontent.com/38295612/107875789-b9f57680-6e76-11eb-832a-2fd136d9e9c1.png)

Chrome Output (iPhone X Preview)
![image](https://user-images.githubusercontent.com/38295612/107875797-c8439280-6e76-11eb-9777-83ac7373d3af.png)

Survey-reporting-dashboard-guide output for both Firefox and Chrome after implementing new code to Getting the Most out of GitHub. (iPhone 6/7/8/X Preview)
![image](https://user-images.githubusercontent.com/38295612/107875365-4a7e8780-6e74-11eb-9b64-f87774e95e1c.png)

**Comment by abenipa3** [comment]
Plans Going Forward (Feedback from Dev Meeting on 2/14/21):

- Focus on restructuring Getting the Most Out of Github and Guide Page files (Make it cross-browser compatible).
- Add content from issue #918.
- Create a new issue to fix Survey Reporting Dashboard for consistency.

**Comment by abenipa3** [comment]
Progress: Researched how to resolve cross-browser compatibility issues, but not sure how to proceed. Updated text size and alignment. Pushed my latest code for review as a draft (#933).

"Final" Output for mobile via (Firefox):
![image](https://user-images.githubusercontent.com/38295612/108634652-60211d80-742f-11eb-94af-a96e3abbb125.png)


Blocks: Need a final confirmation to see if restructuring is necessary after my updates in the Getting Started with Github file affected survey-reporting-dashboard-guide on Chrome as seen below. Danielle and I will also go over the codes to investigate today at 3 pm.


Output for survey-reporting-dashboard-guide after reverting back to the original (Susan's) code in Getting Started with Github via Chrome:
![image](https://user-images.githubusercontent.com/38295612/108633793-1afaec80-742b-11eb-9d3e-dbf44e5aa34a.png)

Output for survey-reporting-dashboard-guide after reverting back to the original (Susan's) code in Getting Started with Github via Firefox:
![image](https://user-images.githubusercontent.com/38295612/108635431-3c140b00-7434-11eb-848f-9b87f3c5833d.png)

Output for survey-reporting-dashboard-guide after updating Getting Started with Github with my code via Firefox:
![image](https://user-images.githubusercontent.com/38295612/108635087-10902100-7432-11eb-862a-3f0f2606447f.png)

Availability: Open.

ETA: GitHub Issues 2/27/21. Wiki Page TBD.

**Comment by akibrhast** [comment]
### Prospective Resource Link
https://jekyllrb.com/docs/step-by-step/04-layouts/

**Comment by abenipa3** [comment]
Researching Format Guide Pages Parse with Markdown. #1085
